cypress/integration/part1.js

```diff
@@ -0,0 +1,59 @@
+describe('Part 1', () => {
+
+    it('6', () => {
+        // "Automatyzujemy stronę Datepicker - wpisujemy date i sprawdzamy czy została wybrana poprawna"
+        // w tym datepickerze nie da rady pisać z palca... musiałem wymyślić coś innego do zrobienia
+
+        cy.visit('https://webdriveruniversity.com/Datepicker/index.html')
+
+        let date = new Date()
+        const today = date.getDate()
+        const tomorrow = today + 1
+        cy.log(date.toString())
+        cy.log(tomorrow)
+
+        cy.get('#datepicker input').click()
+        cy.get('[class="today day"]').should('have.text', today.toString())
+        cy.get('[class="today day"]').siblings().contains('td', tomorrow.toString()).click()
+        cy.get('#datepicker input').click()
+        cy.get('[class="active day"]').should('have.text', tomorrow.toString())
+    })
+
+
+    it('7', () => {
+        // Automatyzujemy stronę Autocomplete TextField - wpisujemy 3 pierwsze znaki i wybieramy 2 element z listy podpowiadanej np. ('chi')
+
+        cy.visit('https://webdriveruniversity.com/Autocomplete-TextField/autocomplete-textfield.html')
+
+        cy.get('#myInput').type('Chi')
+        cy.get('#myInputautocomplete-list div').eq(1).click()
+    })
+
+    before(() => {
+        // the button appears after 5 seconds but set 20 to be safe
+        Cypress.config('defaultCommandTimeout', 20000)
```

Comment:
przekazanie timeoutu do clicka nie wystarcza :)?

Reply:
otóż nie, jak robię tak jak poniżej to nie działa

```suggestion
        cy.get('#myDiv[style="display: block;"] p').click({timeout: 20000})
```

ale pewnie coś źle robię

Reply:
faktycznie, tak działa
`cy.get(AjaxLoader.greenButton, {timeout: 20000}).click()`